app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1512,6 +1538,55 @@ class BrowserTabFragment :
         }
     }
 
+    private fun configureWebViewForAutofill(it: DuckDuckGoWebView) {
```

Comment:
Nit: Could it be worth moving this logic somewhere else outside of the BroweserTabFragment? i find this class already big and confusing so it would be good to not add more things here.

Reply:
I've reduced some of what's required in here since you last reviewed, introducing some dedicated classes to process the results.

We're still going to need to add some code to manage displaying the dialogs, i think, but agree on the goal and will keep working to reduce further additions to BTF.